pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -241,9 +241,11 @@ stdenv.mkDerivation (finalAttrs: {
       ./0016-systemctl-edit-suggest-systemdctl-edit-runtime-on-sy.patch
       ./0017-meson.build-do-not-create-systemdstatedir.patch
       ./0018-Revert-bootctl-update-list-remove-all-instances-of-s.patch # https://github.com/systemd/systemd/issues/33392
+      ./0019-meson-Don-t-link-ssh-dropins.patch
```

Comment:
Could you briefly explain the reason for this patch in a comment or such? Why do we deviate from upstream here?

Reply:
sure! I added a comment.